scripts/pre_commit_linter.py

```diff
@@ -505,6 +541,27 @@ def _check_newline_character(all_files):
     return summary_messages
 
 
+def _check_bad_pattern_in_file(filename, content, pattern):
+    """
+        Detects whether the given pattern is present in the file.
```

Comment:
Formatting:
```
"""Detects whether the given pattern is present in the file.

Args:
    filename: str. Name of the file.
    ...
"""
```

Also: Missing returns docstring?

Reply:
done. 

Ah, I forgot the format. That's an indication that it's been long I contributed anything here :stuck_out_tongue: